app/logic/events.py

```diff
@@ -171,16 +172,16 @@ def getOtherEvents(term):
     # Gets all events that are not associated with a program and are not trainings
     # Gets all events that have a program but don't fit anywhere
     otherEvents = list(Event.select(Event, Program)
-                    .join(ProgramEvent, JOIN.LEFT_OUTER)
-                    .join(Program, JOIN.LEFT_OUTER)
-                    .where(Event.term == term,
-                           Event.isTraining == False,
-                           Event.isAllVolunteerTraining == False,
-                           ((ProgramEvent.program == None) |
-                            (Program.isStudentLed == False) &
-                            (Program.isBonnerScholars == False)))
-                    .order_by(Event.id)
-                  )
+                        .join(ProgramEvent, JOIN.LEFT_OUTER)
+                        .join(Program, JOIN.LEFT_OUTER)
+                        .where(Event.term == term,
+                               Event.isTraining == False,
+                               Event.isAllVolunteerTraining == False,
+                               ((ProgramEvent.program == None) |
+                                (Program.isStudentLed == False) &
+                                (Program.isBonnerScholars == False)))
```

Comment:
We should put parentheses around the `Program.` pair so we are explicit about the order.